dll/win32/shell32/shfldr.h

```diff
@@ -47,11 +47,54 @@ For Details See:
 #define SHFSF_COL_FATTS         4 // File attributes
 #define SHFSF_COL_COMMENT       5
 
+#define MAXREQUIREDREGITEMS 3
```

Comment:
Is my browser broken or is this `MAXREQUIREDREGITEMS` not used in the PR?

Reply:
No you are correct. I was going to implement caching of the registry attributes for these items but stopped to prevent the PR from getting too big.